docs/doc_builder.py

```diff
@@ -1331,7 +1331,7 @@ def run(project_path, temp_directory, *doc_paths):
 
             if html_includes:
                 html_includes = list(
-                    ':ref:`{0}`\n'.format(inc)
+                    ':ref:`{0}`_h\n'.format(inc)
```

Comment:
```suggestion
                    ':ref:`{0}_h`\n'.format(inc)
```

Reply:
Hi, @liamHowatt .

I assume the question is "why was this done?"  As you may be familiar, in .RST files, hyperlinks are created in several ways.  The one that this change affects is links to the .H files containing the API details.  During docs generation, an .RST file is generated for for `sphinx` for the API pages, let's say `lv_obj.h`.  At the top of it is a hyperlink anchor :
```
.. _lv_obj_h:

========
lv_obj.h
========
```
and then `breathe` steps in and inserts the API-page content while processing this file.

That changes the name of this anchor such that if documentation needs to refer to API pages, they now include a "_h" at the end of their hyperlink reference.  And this occurs at the bottom of files like `./docs/libs/gif.rst`.

What it accomplishes is the elimination of about 30 `sphinx` warnings (`duplicate label`) that were occurring during docs generation, due to the anchor name `gifdec` already occurring elsewhere, but it is required to be unique to the entire set of docs.  Each of those warnings meant that a hyperlink was not being created, but a warning message instead.

I hope that answers your question.

Kind regards,
Vic